shared/src/business/useCaseHelper/docketEntry/fileDocumentOnOneCase.js

```diff
@@ -0,0 +1,133 @@
+const {
+  aggregatePartiesForService,
+} = require('../../utilities/aggregatePartiesForService');
+const {
+  ENTERED_AND_SERVED_EVENT_CODES,
+} = require('../../entities/courtIssuedDocument/CourtIssuedDocumentConstants');
+const { Case } = require('../../entities/cases/Case');
+const { DOCKET_SECTION } = require('../../entities/EntityConstants');
+const { WorkItem } = require('../../entities/WorkItem');
+
+exports.fileDocumentOnOneCase = async ({
```

Comment:
First off, I love how this is brought into a useCaseHelper. Smaller, easier to test, and perhaps other functions can make use of it and/or it can be broken up further. Big win!

Secondly, maybe consider a name change to `fileAndServeDocumentOnOneCase` since it's doing both? 

The work down in `shared/src/business/useCases/docketEntry/addPaperFilingInteractor.ts` made me wish there  was just a `fileDocumentOnOneCase`.

Reply:
Thank you, and on it!